<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Login</class>
 <widget class="QWidget" name="Form_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="Btn_Cancel">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>220</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_Login">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEd_UserName">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEd_UserPassword">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>权限：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cBx_permission">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>管理员</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>工程师</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>操作员</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="Btn_LoginEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>250</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>92</y>
         <width>54</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEd_UserNameEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEd_UserPasswordEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>42</y>
         <width>54</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Btn_CancelEdit">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>250</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LineEd_UserPasswordEditC">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>140</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>113</x>
         <y>192</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>权限：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="CBox_UserLevelEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>231</width>
         <height>31</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>管理员</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>工程师</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>操作者</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
